stock.go

```diff
@@ -78,10 +78,29 @@ func NewCrypto(ticker string, token string, name string, nickname bool, color bo
 func (s *Stock) Shutdown() {
 	s.close <- 1
 }
+// THIS FUNCTION IS THE SAME FOR STOCK AND CRYPTO, IF YOU WANT DIFFERENT RESPONSES, MAKE A DIFFEREN'T FUNCTION FOR EACH AND CHANGE THE LINES: 142 AND 340 TO THEIR NEW FUNCTION NAME
+func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
+    //ignores itself
+    if m.Author.ID == s.State.User.ID {
+        return
+    }
+    //listens for mention
+    for _, user := range m.Mentions {
+        if user.ID == s.State.User.ID {
+            s.ChannelMessageSend(m.ChannelID, "Bot  online;") 
+        }
+    }
+        
+    // the xxxxx should be what the bot responds to, with the 'response here' being the response
+    if m.Content == "xxxxx" {
+        s.ChannelMessageSend(m.ChannelID, "Response here*")
+    }
```

Comment:
Lets remove this for now if we dont know what to put. Can add back later if we want.

Reply:
Yea thats fine, was just a 'format' for anyone wanting to add it. Ill // it out?